internal/providers/terraform/azure/registry.go

```diff
@@ -137,6 +137,7 @@ var ResourceRegistry []*schema.RegistryItem = []*schema.RegistryItem{
 	getApplicationInsightsStandardWebTestRegistryItem(),
 	getRecoveryServicesVaultRegistryItem(),
 	getBackupProtectedVmRegistryItem(),
+	getStorageManagementPolicy(),
```

Comment:
I'm okay dropping `RegisterItem` part, but was it intended? :)

Reply:
oops no i'll fix that